packages/module/patternfly-docs/content/examples/catalogTile.css

```diff
@@ -17,11 +17,11 @@
   font-size: small;
 }
 .ws-react-e-catalog-tile .catalog-tile-pf-header .catalog-tile-pf-subtitle a {
-  color: #0066CC;
+  color: var(--pf-t--global--border--color--brand--default);
   text-decoration: none;
 }
 .ws-react-e-catalog-tile .catalog-tile-pf-header .catalog-tile-pf-subtitle a:hover {
-  color: #004080;
+  color: var(--pf-t--global--border--color--brand--hover);
```

Comment:
This is the old link hover color, so you can use `--pf-t--global--text--color--link--hover`